src/Batch/Batch/Batch.csproj

```diff
@@ -26,6 +26,12 @@
     </Compile>
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Update="Azure.Core" Version="1.48.0" />
```

Comment:
Version of Azure.Core is determined by https://github.com/isra-fel/azure-powershell/blob/99c8143a222f965c0b98b09c95a21bfad23986f5/tools/Common.Netcore.Dependencies.targets#L25

Assigning different version may cause runtime errors to the module (if it depends on anything in the newer version).

Will update in a separate PR